BoxContentSDK/BoxContentSDK/OAuth2/BOXOAuth2Session.m

```diff
@@ -18,6 +18,7 @@
 #import "BOXUser_Private.h"
 #import "NSDate+BOXContentSDKAdditions.h"
 #import "BOXAbstractSession_Private.h"
+#import "BOXContentClient.h"
```

Comment:
What are your thoughts on this as a reason to pass in the URL/s to use?

Instead of querying up into the content client - it's given what it needs only...